codewars/cardbuilder.go

```diff
@@ -25,22 +32,41 @@ func ConstructCard(settings url.Values, user User) (template string, err error)
 	if settings.Get("top_languages") == "true" {
 		template = SetIcons(template, user.Ranks.Languages)
 	}
-	stroke := ""
-	if settings.Get("stroke") != "" {
-		stroke = "stroke: " + settings.Get("stroke")
-	}
 	template = strings.Replace(template, "{rankName}", user.Ranks.Overall.Name, 1)
 	template = strings.Replace(template, "{clan}", user.Clan, 1)
 	template = strings.Replace(template, "{leaderboardPosition}", strconv.Itoa(user.LeaderboardPosition), 1)
 	template = strings.Replace(template, "{honor}", strconv.Itoa(user.Honor), 1)
 	template = strings.Replace(template, "{score}", strconv.Itoa(user.Ranks.Overall.Score), 1)
-	template = strings.Replace(template, "{rankColor}", LevelColors[user.Ranks.Overall.Name], -1)
 	template = strings.Replace(template, "{totalCompleted}", strconv.Itoa(user.CodeChallenges.TotalCompleted), 1)
-	template = strings.Replace(template, "{strokeColor}", stroke, 1)
-	template = strings.Replace(template, "{cardColor}", Themes[theme].Card, -1)
-	template = strings.Replace(template, "{headlineFontColor}", Themes[theme].Headline_font, -1)
-	template = strings.Replace(template, "{bodyFontColor}", Themes[theme].Body_font, -1)
-	template = strings.Replace(template, "{badgeColor}", Themes[theme].Rank_badge, -1)
-	template = strings.Replace(template, "{iconColor}", Themes[theme].Icon, -1)
+
+	// using HTML templates
+	templ, err := templatePkg.New("svg").Parse(template)
+	if err != nil {
+		fmt.Printf("error creating template: %v\n", err)
+		return
+	}
+
+	theme := Themes["default"]
+	if settings.Get("theme") != "" {
+		theme = Themes["theme"]
```

Comment:
I think this should be something like:
```go
theme := Themes["default"]
if settings.Get("theme") != "" {
	theme = Themes[settings.Get("theme")]
}
```
otherwise only the `default` Theme would work.

Reply:
ops, fixed! 

Thank you ;)